cekit/builders/podman.py

```diff
@@ -25,8 +26,12 @@ def dependencies(params=None) -> DependencyDefinition:
 
     def run(self):
         """Build container image using podman."""
+
+        podman = shutil.which("podman")
+        if podman is None:
+            raise RuntimeError("podman binary was not found int he system.")
```

Comment:
Typo : 
```suggestion
            raise RuntimeError("podman binary was not found in the system.")
```

Reply:
probably I didn't rebase and force pushed at some point.

Reply:
updated.